source/globals.d.ts

```diff
@@ -61,3 +61,6 @@ declare namespace JSX {
 interface NamedNodeMap {
 	[key: string]: Attr;
 }
+
+
+declare module 'size-plugin';
```

Comment:
I'll check tonight why this isn't working,

Reply:
So it turns out that ts-node, which webpack is using under the hood to process the webpack.config.ts, is not able to detect what is set in the globals.d.ts. So what you can do is add a `paths` setting in the tsconfig.json and create a d.ts file for the `size-plugin` package. However, [ts-node is unable to read the paths property in tsconfig.json](https://github.com/TypeStrong/ts-node/issues/138#issuecomment-349222070). To be able to read `paths`, you need to install the [tsconfig-paths-webpack-plugin](https://github.com/dividab/tsconfig-paths-webpack-plugin). This seems like a lot of work just to support missing types of one package. I'd suggest we stay with the disable eslint rules for now. Thoughts @sindresorhus @bfred-it?

Reply:
`// @tsignore`?

Reply:
Yeah, I think that's a good middle ground until the `size-plugin` has types.

Reply:
@nickytonline Just a heads up, `ts-node` does support types for the compiler. That issue, in particular, is more about resolving executable code based on the `paths` config (e.g. like what you can do in Webpack by aliases). If you wanted you could write a definition and it would work.

Reply:
@blakeembrey how to do that in this case? `./source` is defined in `tsconfig` but why isn't it automatically picked up by ts-node?

Reply:
@bfred-it `ts-node` does not use `files`/`include`/`exclude` by default, but it does support `paths` and `typeRoots` by default. However, you can use them by including `--files`. This is mainly because `ts-node` is used in dozens of different ways and by different use-cases, so excluding by default resulted in the right trade-off between performance and flexibility.

Reply:
I mainly commented because of this:

> However, ts-node is unable to read the paths property in tsconfig.json. To be able to read paths, you need to install the tsconfig-paths-webpack-plugin.

This is a misunderstanding in that thread. `paths` is supported by `ts-node` in the same way TypeScript supports it. It does not, which most people want, automatically rewrite your imports to do things like `@myapp` to automatically come from `./src`.